docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header divider="line">Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane divider="line">Pane</PageLayout.Pane>
+  <PageLayout.Footer divider="line">Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+## Props
+
+### PageLayout
+
+<!-- TODO: Resoponsive variants -->
+
+<PropsTable>
+  <PropsTableRow
+    name="containerWidth"
```

Comment:
Thoughts on dropping the `container` and calling it just `width` as you've attached it on the root of the component? 

```jsx
<PageLayout width="large">
  <PageLayout.Header>Header</PageLayout.Header>
  <PageLayout.Content>Content</PageLayout.Content>
  <PageLayout.Pane>Pane</PageLayout.Pane>
  <PageLayout.Footer>Footer</PageLayout.Footer>
</PageLayout>
```

Reply:
`width` might be a little confusing because the "width" of `PageLayout` is always 100%. The width of the container that is nested inside `PageLayout` is what `containerWidth` controls.

![image](https://user-images.githubusercontent.com/4608155/152020934-a0afe764-8965-4925-b161-a4e28a10b351.png)